packages/ckeditor5-fullscreen/tests/handlers/abstracteditor.js

```diff
@@ -3,32 +3,62 @@
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-licensing-options
  */
 
+/* globals document */
+
 import global from '@ckeditor/ckeditor5-utils/src/dom/global.js';
 import { ClassicEditor } from '@ckeditor/ckeditor5-editor-classic';
 import { Paragraph } from '@ckeditor/ckeditor5-paragraph';
 import { Essentials } from '@ckeditor/ckeditor5-essentials';
+import { PresenceListUI, PresenceList } from '@ckeditor/ckeditor5-real-time-collaboration';
+import { DocumentOutline, DocumentOutlineUI } from '@ckeditor/ckeditor5-document-outline';
+import mockCloudServices, { CloudServicesMock } from '@ckeditor/ckeditor5-real-time-collaboration/tests/_utils/mockcloudservices.js';
```

Comment:
There's a problem with using the commercial features in the open source repository. We decided to use `/* istanbul ignore next -- @preserve */` to prevent CC to fall below 100%, but provide the coverage on the premium packages side. So e.g. in document outline package there should be a test with fullscreen loaded that checks if everything is ok.